### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by Sandor-B**
For assignment 3 of SF-1 - Sandor Balogh

### CODE REVIEW

File: exponential_list_check/io.json
Comment (tkazi0) [inline exponential_list_check/io.json:1]: JSON is valid, inputs and outputs are appropriate.

File: exponential_list_check/meta.json
Comment (tkazi0) [inline exponential_list_check/meta.json:1]: No issues here

File: exponential_list_check/starter.py
Comment (tkazi0) [inline exponential_list_check/starter.py:1]: Type annotation is well added, considering the use of lists.

**Review by tkazi0 — APPROVED** [review]
Everything looks good.

**Comment by emaj9** [comment]
nice

**Pull request merged**